0_web_functions.R

```diff
@@ -86,14 +86,16 @@ set_web_parameters <- function(file_path) {
 set_portfolio_parameters <- function(file_path) {
   cfg <- config::get(file = file_path)
 
-  portfolio_name_ref <<- cfg$parameters$portfolio_name_ref
-  portfolio_name_in <<- cfg$parameters$portfolio_name_in
-  investor_name_in <<- cfg$parameters$investor_name_in
+  portfolio_name <<- cfg$parameters$portfolio_name
+  investor_name <<- cfg$parameters$investor_name
+  peer_group <<- cfg$parameters$peer_group
+  language_select <<- cfg$parameters$language
+  user_id <<- cfg$parameters$user_id
 }
 
 add_naming_to_portfolio <- function(portfolio_raw) {
-  portfolio_raw$portfolio_name <- portfolio_name_in
-  portfolio_raw$investor_name <- investor_name_in
+  portfolio_raw$portfolio_name <- portfolio_name
+  portfolio_raw$investor_name <- investor_name
```

Comment:
this update requires merging the stress test PR: https://github.com/2DegreesInvesting/StressTestingModelDev/pull/40
@maurolepore @cjyetman